examples/gfx/opengl/3_opengl_3d_cube/main.cpp

```diff
@@ -86,7 +86,7 @@ int main() {
     shader_program.bind_buffer("Camera", 1, camera_ubo);
     auto camera_ptr = static_cast<Camera*>(camera_ubo.map());
         camera_ptr->proj = Mat4f::perspective(1, window->get_aspect(), 0.1f, 100.0f);
-        camera_ptr->view = Mat4f::translation(Vec3f(0, 0, 5));
+        camera_ptr->view = Mat4f::view(Vec3f(0.0f, 0.0f, 10.0f), Vec3f::back, Vec3f::down);

```

Comment:
Just a note

Actually ``Mat4f::translation(Vec3f(0, 0, 5));`` is ok, there is no relly differenrce comapring with ``Mat4f::view(Vec3f(0.0f, 0.0f, 10.0f), Vec3f::back, Vec3f::down);``

Actually it should be Mat4f::view(Vec3f(0.0f, 0.0f, -10.0f), Vec3f::forward, Vec3f::up);
Third param of Mat4f::view is UP vector, so it is more reasonable to use **up** instead of **down**
``Vec3f(0.0f, 0.0f, 10.0f), Vec3f::back`` actually good, but I think that it should make more sence if camera is in front of object and not behind.

Also, I see. You probably have taken this line from Vulkan examples(about third param). Vulkan renderer everything upside down comparing with OpenGL, so ya I just flipped camera to compensate this.

Reply:
Ok, I understand. I will fix it then